<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QWidget" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#password_label {
	color: white;
	font-size: 20px;
}</string>
   </property>
   <property name="text">
    <string>Member ID(password):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="header_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>601</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>32</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#header_label {
	font-family: Arial Narrow;
	color: black;
	font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Bulk Club Registration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLE">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#password_textbox {
	color: white;
	background: black;
	font-family: arial;
	font-siz	e: 16px;
	border:1px solid #ffffff;
	border-radius: 2px;
}

QLineEdit#password_textbox:hover {
	background: #191b1c;
	border: 2px solid white;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCB">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox#user_type_combo{
	color: white;
	font-family: arial;
	background: pale;
	border:1px solid #ffffff;
	border-radius: 2px;
}

QComboBox#user_type_combo:hover {
	background: #191b1c;
	border: 2px solid white;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Executive</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Regular</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#username_label {
	font-size: 20px;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#login_button{
	background: gray;
	color: black;
	font-size: 16px;
	border:2px solid #000000;
	border-radius: 5px;
}

QPushButton#login_button:hover{
	background: #b2b9bf;
}</string>
   </property>
   <property name="text">
    <string>Register Now</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLE">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#username_textbox {
	color: white;
	background: black;
	font-family: arial;
	font-siz	e: 16px;
	border:1px solid #ffffff;
	border-radius: 2px;
}

QLineEdit#username_textbox:hover {
	background: #191b1c;
	border: 2px solid white;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#user_type_label {
	font-size: 18px;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>Type of User:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#confirm_password_label {
	color: white;
	font-size: 20px;
}</string>
   </property>
   <property name="text">
    <string>Date (MM/DD/YYYY):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLE">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#confirm_password_textbox {
	color: white;
	background: black;
	font-family: arial;
	font-siz	e: 16px;
	border:1px solid #ffffff;
	border-radius: 2px;
}

QLineEdit#confirm_password_textbox:hover {
	background: #191b1c;
	border: 2px solid white;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
